### ISSUE TITLE
Wrong text domain?

### ISSUE THREAD

**Issue by mrleemon** [woocommerce/action-scheduler#377]
In the following files:

```
includes/libraries/action-scheduler/classes/ActionScheduler_ListTable.php
includes/libraries/action-scheduler/classes/ActionScheduler_Logger.php
includes/libraries/action-scheduler/classes/ActionScheduler_WPCLI_QueueRunner.php
includes/libraries/action-scheduler/classes/ActionScheduler_wcSystemStatus.php
includes/libraries/action-scheduler/classes/ActionScheduler_Store.php
includes/libraries/action-scheduler/classes/ActionScheduler_WPCLI_Scheduler_command.php
includes/libraries/action-scheduler/classes/ActionScheduler_wpPostStore.php
includes/libraries/action-scheduler/classes/ActionScheduler_wpPostStore_PostTypeRegistrar.php
includes/libraries/action-scheduler/classes/ActionScheduler_wpPostStore_TaxonomyRegistrar.php
includes/libraries/action-scheduler/classes/ActionScheduler_InvalidActionException.php

```

the text domain used is **action-scheduler** instead of  **woocommerce**. Therefore, the "Scheduled Actions" tab in the "Status" section remains unstranslated when using WooCommerce in other languages other than English.

**Comment by claudiosanches** [comment]
I moved this issue to the Action Scheduler repository, because there's several changes that need to be made in this plugin in order to proper handle text domain on WooCommerce.

Also I think that we should start implement it on WooCommerce as package like we do for Product Blocks and REST API.

**Comment by thenbrent** [comment]
> start implement it on WooCommerce as package like we do for Product Blocks and REST API

@claudiosanches does that allow us to use the `woocommerce` text domain?

As Action Scheduler is used as a library in various plugins (and at least one theme), I've never really been sure what to do about text domain. We can't be sure any plugin will be active on a site using it.

**Comment by claudiosanches** [comment]
@thenbrent we should use `action-scheduler` as text domain.
On WooCommerce we update the text domain to `woocommerce` while installing the package:

https://github.com/woocommerce/woocommerce/blob/master/bin/package-update.sh#L32-L33

But for now is missing text domains in several locations, also missing note for translators, and missing to follow some standards for translations.

**Comment by claudiosanches** [comment]
There the Pull Request: #382

**Comment by thenbrent** [comment]
Fixed with #382.
